test/Xpring/xpring-client.test.ts

```diff
@@ -27,6 +27,12 @@ const fakedEnableDepositAuthValue = new TransactionResult(
   true,
   true,
 )
+const fakedAuthorizeSendingAccountValue = new TransactionResult(
```

Comment:
this is identical to the `fakedEnableDepositAuthValue` - maybe we should just rename that to something more generic and use our successful `TransactionResult` everywhere it's called for. (since I know you've got another PR a lot like this one coming)

Reply:
i'll give it a shot!

Reply:
i went for this change